client/my-sites/plans/current-plan/jetpack-checklist/index.tsx

```diff
@@ -115,18 +119,45 @@ class JetpackChecklist extends PureComponent< Props > {
 					completedTitle={ task.completedTitle }
 					description={ task.description }
 					duration={ task.duration }
+					firstIncomplete={ this.getFirstIncompleteTask( taskIds ) }
```

Comment:
Similar as https://github.com/Automattic/wp-calypso/pull/33666/files#r292552161. The `Task` component shouldn't need to care what's the first incomplete task, that's the `Checklist'`s job.